packages/core-flows/src/order/steps/set-tax-lines-for-items.ts

```diff
@@ -0,0 +1,128 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import {
+  CreateLineItemTaxLineDTO,
+  CreateShippingMethodTaxLineDTO,
+  IOrderModuleService,
+  ItemTaxLineDTO,
+  OrderDTO,
+  ShippingTaxLineDTO,
+} from "@medusajs/types"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+
+interface StepInput {
+  order: OrderDTO
```

Comment:
suggestion: We are only using the id, should we update the type to reflect that and not force to pass an entire order dto?

Reply:
Same suggestion in general for all inputs, the idea being that we should ask for what we need and not more to prevent forcing the consumer to retrieve all the data that we do not need